uninstaller.nsh

```diff
@@ -1,16 +1,32 @@
+!define CLI_VERSION "1.5.3"
```

Comment:
Is there a way we can put the version number in the package.json and read it here?

Reply:
@JamieRuderman it's not posible

Reply:
Okay just thought I'd check!  :)